app/components/works/contributor_row_component.html.erb

```diff
@@ -39,23 +40,24 @@
             <%= form.label :with_orcid, "Enter ORCID iD", value: "true", class: "form-check-label fw-semibold" %>
           </div>
           <div class="row person-identifier-row" data-contributors-target="personOrcid">
+            <% text_field_id_prefix = "work_#{namespace}_orcid_#{form.options[:child_index]}_" %>
             <div class="col">
-              <%= form.label :orcid, orcid_label, class: "form-label" %>
+              <%= form.label :orcid, orcid_label, for: "#{text_field_id_prefix}id", class: "form-label" %>
```

Comment:
"for" hooks binds this label to the text field, and the expression ensures that each contributor has unique labels.